### PR TITLE
Don't generate methods and setup linting CI

### PR DESCRIPTION
**Pull request by rchl**
I've changed it so that the `generate.py` itself decides where to insert empty lines as it's easier to control spacing between different entities that way.

Also set up linting CI.

### CODE REVIEW

File: generate.py [hunk 1]
```diff
@@ -14,25 +14,29 @@ def generate() -> None:
...
+            "# Code generated. DO NOT EDIT.",
+            f"# LSP v{specification_version}\n",
+            "from typing_extensions import NotRequired",
+            "from typing import Dict, List, Literal, TypedDict, Union, Tuple",
+            "from enum import Enum\n\n",
+            "URI = str",
+            "DocumentUri = str",
+            "Uint = int",
+            "RegExp = str",
+            # generate_requests(lsp_json['requests'])
```
Comment (predragnikolic) [inline generate.py:27]: This comment can be removed

File: generate.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 from lsp_schema import MetaModel
 from utils.generate_enumerations import generate_enumerations
-from utils.generate_requests import generate_requests
+# from utils.generate_requests import generate_requests
```
Comment (predragnikolic) [inline generate.py:6]: Generate request can be deleted,
As it is not needed now.
What do you think?

Comment (rchl) [reply generate.py:6]: sure, can clean up

**Review by predragnikolic — APPROVED** [review]

File: generate.py (hunk 2 above)
Comment (predragnikolic) [reply generate.py]: Thanks :⁠-⁠)

**Pull request merged**